docs/docs/getting-started.server.mdx

```diff
@@ -481,6 +481,42 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 
 ### Site generators
 
+#### Astro
+
+[Astro](https://astro.build/) has an [official MDX integration](https://docs.astro.build/guides/integrations-guide/mdx/).
+Astro recommends using [the `astro add` CLI](https://docs.astro.build/en/guides/integrations-guide/#automatic-integration-setup)
+to install dependencies and update your config automatically:
+
+```sh
+# Using npm
+npx astro add mdx
+# Using yarn
+yarn astro add mdx
+# Using pnpm
+pnpx astro add mdx
```

Comment:
```suggestion
npx astro add mdx
```

I’d prefer just showing `npx`: a) everyone has it, b) most users use it, c) people who use yarn/pnpm most likely know how to use their alternative, d) very new beginners might not be aware of yarn/pnpm and think all 3 commands are needed, e) consistency with rest of install sections on the website

If you feel strongly about this and this is how Astro documents all install sections, I am open to keep all three.

Reply:
Fair enough, Astro had these thoughts too! This _is_ the pattern we use across the Astro docs, but I don't feel too strongly about it. I'm happy to use `npx` alone here 👍